editor.planx.uk/src/@planx/components/Pay/Editor/FeeBreakdownSection.tsx

```diff
@@ -1,13 +1,134 @@
+import HelpIcon from "@mui/icons-material/Help";
 import ReceiptLongIcon from "@mui/icons-material/ReceiptLong";
+import Box from "@mui/material/Box";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Tooltip from "@mui/material/Tooltip";
+import Typography from "@mui/material/Typography";
+import { BoldTableRow } from "@planx/components/shared/Table/TableRow";
 import { useFormikContext } from "formik";
 import { hasFeatureFlag } from "lib/featureFlags";
-import React from "react";
+import React, { PropsWithChildren } from "react";
 import ModalSection from "ui/editor/ModalSection";
 import ModalSectionContent from "ui/editor/ModalSectionContent";
 import InputRow from "ui/shared/InputRow";
 import { Switch } from "ui/shared/Switch";
 
 import { Pay } from "../model";
+import { VAT_RATE } from "../Public/FeeBreakdown";
+
+const DataField: React.FC<PropsWithChildren> = ({ children }) => (
+  <Typography variant="data" sx={(theme) => ({ fontSize: theme.typography.body3.fontSize, pr: 2 })}>{children}</Typography>
+);
+
+const HelpTooltip: React.FC<{ text: string }> = ({ text }) => (
+  <Tooltip title={text} placement="bottom-end">
+    <HelpIcon color="primary"/>
+  </Tooltip>
+)
+
+const ExampleTable: React.FC = () => (
+  <TableContainer>
+    <Table sx={{ [`& .${tableCellClasses.root}`]: { px: 0 } }} >
+      <TableHead>
+        <BoldTableRow>
+          <TableCell>Description</TableCell>
+          <TableCell align="right">Passport field</TableCell>
+        </BoldTableRow>
+      </TableHead>
+      <TableBody>
+        <TableRow>
+          <TableCell>Application fee</TableCell>
+          <TableCell align="right" sx={{ display: "flex", alignItems: "center"}}>
+            <HelpTooltip text="The fee will be the calculated amount (if set), or the payable amount" />
+            <DataField>£ application.fee.calculated</DataField>
+          </TableCell>
+        </TableRow>
+        <TableRow>
+          <TableCell>Reduction</TableCell>
+          <TableCell align="right" sx={{ display: "flex", alignItems: "center" }}>
+            <HelpTooltip text="This is the sum of all reductions applied to the application fee. The list of reductions will be listed below" />
+            <DataField>- £ (calculated - payable)</DataField>
```

Comment:
I'm not sure if this shorthand for `application.fee.calculated - application.fee.payable` is that clear?

The "full" version pretty much always overflows and feels less clear even though the above is not technically correct.

<img width="442" alt="image" src="https://github.com/user-attachments/assets/b6c1f41d-8980-4434-b0ce-09576ae70f05">